xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/schema.xml

```diff
@@ -840,12 +840,14 @@
     </fieldType>
 
     <!-- Czech -->
-    <fieldType name="text_cz" class="solr.TextField" positionIncrementGap="100">
+    <fieldType name="text_cs" class="solr.TextField" positionIncrementGap="100">
       <analyzer> 
         <tokenizer class="solr.StandardTokenizerFactory"/>
         <filter class="solr.LowerCaseFilterFactory"/>
         <filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_cz.txt" />
```

Comment:
OK, I agree.

FTR, here are the upstream issues to follow for anyone interested:
https://issues.apache.org/jira/browse/LUCENE-6366
https://issues.apache.org/jira/browse/SOLR-7267

@pabro Please undo the type name change and leave it to "text_cz" as it is currently in Solr, as you`ve mentioned. As soon as it's done, please ping here so we can apply your PR.